pages/common/git-repl.md

```diff
@@ -0,0 +1,21 @@
+# git repl
+
+> Open a Git shell-like command line.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-repl>.
+
+- Open a Git shell-like command line:
+
+`git repl`
+
+- Run a git command whilst in REPL:
+
+`{{command}}`
```

Comment:
We can help saying how to send arguments to the command, and explicitly say `ls` is not the unix `ls`:
```suggestion
`{{git_subcommand}} {{command_arguments}}`
```

Reply:
```suggestion
`{{git_command}} {{command_arguements}}`
```

Reply:
in the `git` page we say `git {{subcommand}}`.
IMO `git_command` means `git {{subcommand}}`, in the other hands `git_subcommamd` sounds to `checkout`, or another subcommand from Git.

Reply:
I also thank that `{{git_subcommand}}` makes more sense here.